Source/gui/SNTNotificationManager.m

```diff
@@ -100,7 +100,20 @@ - (void)queueMessage:(SNTMessageWindowController *)pendingMsg {
 
   // If GUI is in silent mode or if there's already a notification queued for
   // this message, don't do anything else.
-  if ([SNTConfigurator configurator].enableSilentMode) return;
+  if ([SNTConfigurator configurator].enableSilentMode) {
+    // hash bundle with path
+    dispatch_async(self.hashBundleBinariesQueue, ^{
+          SNTBinaryMessageWindowController *wc = (SNTBinaryMessageWindowController *)pendingMsg;
+          [self hashBundleBinariesForEvent:wc.event withProgressHandler:^(NSUInteger progressCount) {
+              // progress is reported to the distributed notification
+              [self postDistributedNotification: pendingMsg withProgress: progressCount];
```

Comment:
Remove the spaces after the colons in method calls

Reply:
fixed with `./Testing/fix.sh`